webgpio/device/redis.py

```diff
@@ -25,7 +25,7 @@
     try:
         gpio = int(kwargs.get('gpio'))
         assert gpio in gpios
```

Comment:
`assert` expressions may be disabled in runtime, so it may be potentially unreliable to use them